src/main/java/Event.java

```diff
@@ -0,0 +1,11 @@
+public class Event extends Task {
+    private final static char badge = 'E';
+
```

Comment:
naming of methods and class generally follow the naming guideline of using verbs for actions and noun for objects